src/app/pages/blog/search-results/use-all-articles.tsx

```diff
@@ -0,0 +1,33 @@
+import {useState, useEffect} from 'react';
```

Comment:
As diffs [here](https://github.com/openstax/os-webview/pull/2676/commits/0b90a1d3de9ab2661be33c9705394cc848875e6a#diff-45eb947d07efdd829903c9934b8d0749200d6fe5ba2e4e231ebb1c6981e07541). The data.subheading change was due to subheading being a required field. Making the variable was just part of the process; I should probably take that back out.